src/partials/orderwindow.html

```diff
@@ -0,0 +1,151 @@
+<div>
+  <h2>YOUR ORDER</h2>
+  <div class="modalscript is-hidden" data-modal>
+    <div class="windowscript">
+      <button class="buttonscript" type="button" data-modal-close>
+        <svg width="28" height="28" class="iconscript">
+          <use href="./images/icons.svg#icon-x"></use>
+        </svg>
+      </button>
+      <div>
```

Comment:
acesta este un formular. asa ca o sa trebuiasca sa fie inclus in <form>. vezi in cerintele sectiunii.